scripts/drone/pipelines/pr.star

```diff
@@ -116,7 +112,15 @@ def pr_pipelines(edition):
         ])
 
     trigger = {
-        'event': ['pull_request',],
+        'event': [
+            'pull_request',
+        ],
+        'paths': {
+            'exclude': [
+                'docs/**',
+                'packages/**',
```

Comment:
I think the `packages/**` should be removed here. We should run the test-pr pipeline if we have changes in the `packages/**` folder.